pkg/sync/v14/data/ClusterRole.rbac.authorization.k8s.io/customer-admin-project.yaml

```diff
@@ -41,3 +41,12 @@ rules:
   - delete
   - patch
   - update
+- apiGroups:
```

Comment:
Don't we still need the apiGroup `- ""`?

Reply:
I'm not sure what the difference is.  What does supplying an empty "" do vs adding the specific api group `image.openshift.io`.  I'll happily add both but unsure as to why.

Reply:
I'm not actually sure what the difference is, trawling through docs, but I noticed this didn't match the other file.